setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+test_dependencies = [
+    "coverage",
+    "pytest",
+    "pytest-cov",
+]
+
+extras = {
+    "testing": test_dependencies,
+}
+
+with open("README.md", "rt") as f:
+    long_description = f.read()
+
+setup(
+    name="looseleaf",
+    use_incremental=True,
+    author="Mozilla Corporation",
+    author_email="fx-data-dev@mozilla.org",
+    description="Runs a thing that analyzes experiments",
+    url="https://github.com/mozilla/penseive",
+    packages=["pensieve", "pensieve.tests"],
+    install_requires=[
+        "attrs",
```

Comment:
I'm guessing we want to pin versions, just to avoid potential unknown issues introduced by an updated version

Reply:
Per the blog post you linked, I think the right way to do this is in a separate requirements.txt. I think we can delay this until we're actually deploying but I could generate one now.

Reply:
I'm okay with doing that in a separate PR